notification-settings/lib/notification_settings/notification_library.rb

```diff
@@ -44,14 +44,15 @@ def status_allows_create?
       end
 
       def settings_allow_create?
-        target.notification_setting.settings.dig(:enabled)
+        target.notification_setting.settings.fetch(:enabled) { true }
```

Comment:
I think this looks a bit cleaner. What do you say?

```suggestion
        target.notification_setting.settings.fetch(:enabled, true)
```

Reply:
Sure, I don't mind it either way :)